exercises/accumulate/accumulate.test.ts

```diff
@@ -1,32 +1,32 @@
 import accumulate from './accumulate'
 
 describe('accumulate()', () => {
-    it('accumulation empty', () => {
-        const accumulator = (e: number) => e * e
-        expect(accumulate([], accumulator)).toEqual([])
-    })
+  it('accumulation empty', () => {
+    const accumulator = (e: number): number => e * e
+    expect(accumulate([], accumulator)).toEqual([])
+  })
 
-    xit('accumulate squares', () => {
-        const accumulator = (n: number) => n * n
-        const result = accumulate([1, 2, 3], accumulator)
-        expect(result).toEqual([1, 4, 9])
-    })
+  it('accumulate squares', () => {
+    const accumulator = (n: number): number => n * n
+    const result = accumulate([1, 2, 3], accumulator)
+    expect(result).toEqual([1, 4, 9])
+  })
 
-    xit('accumulate upcases', () => {
-        const accumulator = (word: string) => word.toUpperCase()
-        const result = accumulate('hello world'.split(/\s/), accumulator)
-        expect(result).toEqual(['HELLO', 'WORLD'])
-    })
+  it('accumulate upcases', () => {
+    const accumulator = (word: string): string => word.toUpperCase()
+    const result = accumulate('hello world'.split(/\s/), accumulator)
+    expect(result).toEqual(['HELLO', 'WORLD'])
+  })
 
-    xit('accumulate reversed strings', () => {
-        const accumulator = (word: string) => word.split('').reverse().join('')
-        const result = accumulate('the quick brown fox etc'.split(/\s/), accumulator)
-        expect(result).toEqual(['eht', 'kciuq', 'nworb', 'xof', 'cte'])
-    })
+  it('accumulate reversed strings', () => {
+    const accumulator = (word: string): string => word.split('').reverse().join('')
+    const result = accumulate('the quick brown fox etc'.split(/\s/), accumulator)
+    expect(result).toEqual(['eht', 'kciuq', 'nworb', 'xof', 'cte'])
+  })
 
-    xit('accumulate recursively', () => {
-        const result = accumulate('a b c'.split(/\s/), (char: string) => accumulate('1 2 3'.split(/\s/), (digit: string) => char + digit))
+  it('accumulate recursively', () => {
```

Comment:
Only the first should be enabled. 
https://github.com/exercism/typescript/issues/88